qemu/tests/cfg/qmp_command.cfg

```diff
@@ -203,3 +203,15 @@
             cmd_return_value = "[{'name': 'rtl8139'}, {'name': 'e1000e'}, {'name': 'vfio-pci'}, {'name': 'qemu-xhci'}, {'name': 'virtio-blk-pci'}]"
             aarch64:
                 cmd_return_value = "[{'name': 'vfio-pci'}, {'name': 'qemu-xhci'}, {'name': 'virtio-blk-pci'}]"
+        - qmp_query-stats_vm:
+            qmp_cmd = "query-stats target=vm"
+            cmd_result_check = "contain"
+            cmd_return_value = "['provider', 'stats', 'name', 'value']"
+        - qmp_query-stats_vcpu:
+            qmp_cmd = "query-stats target=vcpu"
+            cmd_result_check = "contain"
+            cmd_return_value = "['provider', 'stats', 'name', 'value', 'qom-path']"
+        - qmp_query-stats-schemas:
+            qmp_cmd = "query-stats-schemas"
+            cmd_result_check = "contain"
+            cmd_return_value = "['provider', 'stats', 'name', 'exponent', 'type', 'unit', 'target']"
```

Comment:
Do we to add `base` ?

Reply:
ok, we need add 'base'. thanks

Reply:
@yanan-fu , I have updated it, please check it, thanks